docs/source/api_reference/experimental.rst

```diff
@@ -18,6 +18,10 @@ Change-point utilities:
 
 Classification of time-series:
 
+.. note::
```

Comment:
Do we actually need this as we have it in each class already? The same for nns

Reply:
We also has this in `auto`. Ok, let's try to remove this.